index.html

```diff
@@ -1615,18 +1615,57 @@ <h3>Media Types</h3>
 	<section id="sec-building-blocks">
 		<h1>WoT Building Blocks</h1>
 		<p>
-			The Web of Things provides more concrete building blocks that allow to implement the abstract WoT Architecture.
-			As, they are defined in separate documents, this document provides the overview and a summary.
+			The Web of Things provides specific building blocks to allow the implementation of systems that
+                        conform with the abstract WoT Architecture.
+			The specfics of these building blocks are defined in separate documents; 
+                        this document provides an overview and a summary.
 		</p>
-		
-		<p>
 			<figure id="arch-new-building-blocks">
-				<img src="images/arch-new-building-blocks.png" />
-				<figcaption>Building Blocks</figcaption>
+				<img src="images/wot-building-blocks.png" />
+				<figcaption>Architectural Aspects of a Thing</figcaption>
+			</figure>
+		<p>
```

Comment:
the empty protocol boxes look confusing at first sight. Perhaps fill them with Protocol1 .. Protocol n?

Reply:
done